### ISSUE TITLE
Sorting 

### ISSUE THREAD

**Issue by armandg** [michael-lazar/rtv#502]
After pressing `s` ("Show subscribed subreddits"), 

It would be cool if one could sort the subscribed subreddits, with kind of like the same view when reading submissions or comments ("hot", "top", "rising" and so forth).

**Comment by michael-lazar** [comment]
Nice idea, but I don't think that any of those categories (top, rising, etc) apply to subscriptions. From the api endpoint, https://www.reddit.com/dev/api/#GET_subreddits_mine_{where}, it's  not clear that you can request a certain "order" for the results

**Comment by armandg** [comment]
How about getting the subreddits, iterating them and then making sorting possible? Only alphabetically for my part.

**Comment by michael-lazar** [comment]
Yes I think we could do alphabetical sorting, that should be a very simple change
